bench/benchmarks/frame_duration.js

```diff
@@ -6,6 +6,8 @@ var formatNumber = require('../lib/format_number');
 
 var DURATION_MILLISECONDS = 1 * 5000;
 
+window.devicePixelRatio = 1 / 16;
```

Comment:
What is your intention behind this line? (Answer probably best delivered as a `// comment` in the source)